core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/IndexFamily.java

```diff
@@ -115,6 +117,10 @@ public PlatformType getPlatformType() {
     return platformType;
   }
 
+  public boolean getFake() {
```

Comment:
For this project, we use `isX` or `hasXs` for boolean getters. Could this be updated to match?

Reply:
Yeah, I'll do that right now.